cranelift/filetests/filetests/isa/x64/simd-bitwise-compile.clif

```diff
@@ -376,21 +478,106 @@ block0:
 ;   pushq %rbp
 ;   movq %rsp, %rbp
 ; block0: ; offset 0x4
-;   movdqu 0xb4(%rip), %xmm0
-;   movl $1, %r9d
-;   andq $7, %r9
-;   movd %r9d, %xmm5
-;   psrlw %xmm5, %xmm0
-;   leaq 0x1a(%rip), %rsi
-;   shlq $4, %r9
-;   movdqu (%rsi, %r9), %xmm13
-;   pand %xmm13, %xmm0
+;   movdqu 0x34(%rip), %xmm0
+;   psrlw $1, %xmm0
+;   movdqu 0x17(%rip), %xmm3
+;   pand %xmm3, %xmm0
 ;   movq %rbp, %rsp
 ;   popq %rbp
 ;   retq
 ;   addb %al, (%rax)
 ;   addb %al, (%rax)
-;   addb %bh, %bh
+;   addb %al, (%rax)
+;   addb %al, (%rax)
+;   addb %al, (%rax)
+;   addb %al, (%rax)
+;   addb %al, (%rax)
+;   jg 0xb1
+;   jg 0xb3
+;   jg 0xb5
+;   jg 0xb7
+;   jg 0xb9
+;   jg 0xbb
+;   jg 0xbd
+;   jg 0xbf
+;   addb %al, (%rcx)
+;   addb (%rbx), %al
+;   addb $5, %al
```

Comment:
I'll admit I don't know what's happening here. This I believe is the constant pool area and capstone is trying to decode it as instructions, but I don't know why after this change, which should generate a smaller constant pool, is generating more instructions here. I would hazard a guess that capstone automatically stoped beforehand for some reason and now it's keeping-on-going, but I'm not sure.

Reply:
I think this is at least partially the same issue that @elliottt is tackling in #5798. We've discussed ways to entirely quit trying to disassemble constants since we think the machbuffer knows where they are, but one step at a time.

Reply:
Since #5798 is merged, can you try rebasing and see if the filetests look any better now?

Reply:
Nevermind, I see you already did and it looks like it didn't help. I'm going to declare that this is harmless and move on with reviewing the rest of this PR.